examples/client/Locomotion/src/context/bottomSheetContext/index.js

```diff
@@ -12,6 +12,9 @@ export const SNAP_POINT_STATES = {
   [BS_PAGES.SET_LOCATION_ON_MAP]: ['30%'],
   [BS_PAGES.NO_PAYMENT]: ['30%'],
   [BS_PAGES.NOT_IN_TERRITORY]: ['30%'],
+  [BS_PAGES.CONFIRMING_RIDE]: ['30%'],
```

Comment:
make them ['30%', '30%']. i believe theres a bug in the package, its the only way i found to handle it. https://github.com/gorhom/react-native-bottom-sheet/issues/417